go/cmd/aggregate-crls/aggregate-crls_test.go

```diff
@@ -659,3 +660,74 @@ func Test_SaveRevokedCertificateList(t *testing.T) {
 		t.Fatalf("Data should match exactly - expected=[%+v] loaded=[%+v]", expected, fileBytes)
 	}
 }
+
+func Test_IssuingDPMatching(t *testing.T) {
+	// SEQUENCE {
+	//     INTEGER { 1 }
+	//     SEQUENCE {
+	//       # ecdsa-with-SHA256
+	//       OBJECT_IDENTIFIER { 1.2.840.10045.4.3.2 }
+	//     }
+	//     SEQUENCE {
+	//       SET {
+	//         SEQUENCE {
+	//           # commonName
+	//           OBJECT_IDENTIFIER { 2.5.4.3 }
+	//           PrintableString { "Test CRL" }
+	//         }
+	//       }
+	//     }
+	//     UTCTime { "250701160205Z" }
+	//     UTCTime { "250711150204Z" }
+	//     SEQUENCE {}
+	//     [0] {
+	//       SEQUENCE {
+	//         SEQUENCE {
+	//           # issuingDistributionPoint
+	//           OBJECT_IDENTIFIER { 2.5.29.28 }
+	//           BOOLEAN { TRUE }
+	//           OCTET_STRING {
+	//             SEQUENCE {
+	//               [0] {
+	//                 [0] {
+	//                   [6 PRIMITIVE] { "http://example.com/1.crl" }
+	//                 }
+	//               }
+	//             }
+	//           }
+	//         }
+	//       }
+	//     }
+	//   }
+	//   SEQUENCE {
+	//     # ecdsa-with-SHA256
+	//     OBJECT_IDENTIFIER { 1.2.840.10045.4.3.2 }
+	//   }
+	//   BIT_STRING {
+	//     `00`
+	//     SEQUENCE {
+	//       INTEGER { `0000000000000000000000000000000000000000000000000000000000000000` }
+	//       INTEGER { `0000000000000000000000000000000000000000000000000000000000000000` }
+	//     }
+	//   }
+	// }
+	crlBlock, _ := pem.Decode([]byte(`-----BEGIN X509 CRL-----
+MIHLMHQCAQEwCgYIKoZIzj0EAwIwEzERMA8GA1UEAxMIVGVzdCBDUkwXDTI1MDcw
+MTE2MDIwNVoXDTI1MDcxMTE1MDIwNFowAKAuMCwwKgYDVR0cAQH/BCAwHqAcoBqG
+GGh0dHA6Ly9leGFtcGxlLmNvbS8xLmNybDAKBggqhkjOPQQDAgNHADBEAiAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAIgAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAA=
+-----END X509 CRL-----`))
+
+	crl, _ := x509.ParseCertificateListDER(crlBlock.Bytes)
+
+	fetchUrl := "http://example.com/1.crl"
+	if !crlIssuingDPMatches(crl, fetchUrl) {
```

Comment:
Am I missing something? I'm not seeing where this is defined... (is the review UI messing me up?)

Reply:
Whoops, refactored and forgot to update this test.